### PR TITLE
Hashan update (shop names, new room, dog shelter)

### PR DESCRIPTION
**Pull request by Suiziide**
## Checklist

The list below serves as a reminder to make sure, all needed changes were added to the repository. You can add a `x` between the square brackets to tick the items when in edit mode or just tick the checkboxes when not.

If you forgot something, don't worry. Just do the changes on the same branches and they are added automatically to the pull request.

- [x] Added changed map file
- [x] Used a descriptive pull request title
- [x] Wrote a short description of the change in the pull request description (see below). Include area names where possible.
- [x] Added a short sentence summarizing the changes to the `changelog.txt` file
- [x] Increased the number in `version.txt` by 1

## Included changes
Added new seed shop south of lab.
Updated shop names.
Changed room names of what is now the dog shelter

### CODE REVIEW

File: Map/changelog.txt [hunk 1]
```diff
@@ -465,4 +465,5 @@ changelog = {
 	[[Fixed Targossas2 after update, Eastern Reaches]],
 	[[Updated Milabar's estate, added missing shops in Delos, added magi workshops in Ashtan, Hashan and Targossas, art gallery in Ashtan, some sewer fixes in Targossas, updated a few shop names]],
 	[[Added Heartwood Kin estate, Gaia's Temple, and Eleusis resource hub]]
```
Comment (vadi2) [inline Map/changelog.txt]: Tiny detail, need to have a comma here:

```suggestion
	[[Added Heartwood Kin estate, Gaia's Temple, and Eleusis resource hub]],
```

**Review by vadi2 — APPROVED** [review]
Thanks so much for your contribution to the crowdmap :)

File: Map/changelog.txt (hunk 1 above)
Comment (Suiziide) [reply Map/changelog.txt]: Gotcha, missed that, should be fine now

**Review by keneanung — APPROVED** [review]
Thank you very much for your contribution! 🚀

**Pull request merged**